### PR TITLE
adds default fields for global styleguide

### LINKED ISSUE #6: Add default styleguide fields and vertical tabs to Group

### PR DESCRIPTION
**Pull request by markconroy**
Closes #6

### CODE REVIEW

File: config/optional/core.entity_form_display.group.microsite.default.yml [hunk 1]
```diff
@@ -147,11 +221,42 @@ third_party_settings:
         formatter: closed
         description: ''
         required_fields: true
+_core:
+  default_config_hash: Sc3vG_RGmi7xjhph--d3TyJj581WYAqoubf0kKA79Fk
```
Comment (stephen-cox) [inline config/optional/core.entity_form_display.group.microsite.default.yml]: These lines should be removed

File: config/optional/core.entity_view_display.group.microsite.default.yml [hunk 2]
```diff
@@ -37,14 +44,50 @@ dependencies:
     - link
     - options
     - text
+_core:
+  default_config_hash: AYpmwivb0tsbYYmsFPuk6pAuMuHG5J1EkxJ09cErs5w
```
Comment (stephen-cox) [inline config/optional/core.entity_view_display.group.microsite.default.yml]: Remove

Comment (stephen-cox) [inline config/optional/core.entity_view_display.group.microsite.default.yml]: Remove

**Review by stephen-cox — APPROVED** [review]
Largely looks fine to me. Tested and the style guide works well on the group page. When I created a page and added it to the group it didn't pick up the style settings, but that's not an issue here.

A minor issue; a couple of files contain uuid fields that we don't want.

File: config/optional/core.entity_form_display.group.microsite.default.yml (hunk 1 above)
Comment (markconroy) [reply config/optional/core.entity_form_display.group.microsite.default.yml]: oooooh, pots are calling kettles names!!!!!!

**Comment by markconroy** [comment]
@stephen-cox  you happy enough for me to merge this now? (I removed a few more config hashes that were hanging around in this repo).

**Comment by stephen-cox** [comment]
> @stephen-cox you happy enough for me to merge this now? (I removed a few more config hashes that were hanging around in this repo).

Yep - merge it!

**Pull request merged**